File: Assignment_40/Program2.c
/* Write a recursive program which display below pattern.
Output : 1  2  3  4  5 
*/

#include <stdio.h>

void Display()
{
    static int i = 1;

    if(i <= 5)
    {
        printf("%d\t",i);
        i++;
        Display();
    }
    else
    {
        printf("\n");
    }
}

int main()
{
    Display();

    return 0;
}

File: Assignment_49/Program2.java
/* Write a java program which accepts String from user and display below pattern

Input :   Hello
Output:   H  e  l  l  o
          H  e  l  l  *
          H  e  l  *  *
          H  *  *  *  *
 */

 import java.util.*;

class Pattern
{
    public void Pattern(String str)
    {
        int i = 0, j = 0;

        for(i = 0; i < str.length(); i++)
        {
            for(j = 0; j < str.length(); j++)
            {
                if(j < str.length() - i)
                {
                    System.out.print(str.charAt(j)+"\t");
                }
                else
                {
                    System.out.print("*\t");
                }
            }
            System.out.println();
        }
    }
}

class Program2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the String : ");
        String str = sobj.nextLine();

        Pattern pobj = new Pattern();
        pobj.Pattern(str);
    }
}